builtin/gc.c

```diff
@@ -1678,6 +1657,9 @@ static const char *get_frequency(enum schedule_priority schedule)
 	}
 }
 
+static const char *custom_background_config =
+	"-c credential.interactive=false -c cred.askPass=false";
+
```

Comment:
Just a nit.  As an alternative to defining a static string here, you could #define a macro and use that in each of the fprintf() format strings below (like we do for the `PRIuMAX` or `PRItime` strings).